javascript/src/main/components/BasicCourseSearch/CourseSearchCourseStartEndQtr.js

```diff
@@ -11,6 +11,9 @@ const CourseSearchCourseStartEndQtr = ({ setCourseJSON, fetchJSON }) => {
     const handleSubmit = (event) => {
         event.preventDefault();
         fetchJSON(event, { startQuarter, endQuarter, subjectArea, courseNumber, courseSuf}).then((courseJSON) => {
+            if(courseJSON.total === 0){
+                alert("Empty!");
```

Comment:
Please consider addToast instead of alert

Reply:
Implemented for this and all below!